language/move-lang/src/parser/syntax.rs

```diff
@@ -1549,6 +1562,20 @@ fn parse_optional_type_parameters(tokens: &mut Lexer) -> Result<Vec<(Name, Vec<A
     }
 }
 
+fn parse_struct_type_parameters(tokens: &mut Lexer) -> Result<Vec<StructTypeParameter>, Error> {
```

Comment:
Can you please add a "grammar comment" before this function and update the comment before `parse_struct_decl`? We don't have a separate specification of the grammar so we've adopted a convention of documenting it in comments here. The new grammar non-terminal should be called `StructTypeParameters` (to match the `parse_struct_type_parameters` function name that you're introducing here.

Reply:
for sure! noticed the other functions had the comments but didn't pay too much attention.